### ISSUE TITLE
Accessibility issue with VirtualScroll DataTable

### ISSUE THREAD

**Issue by mertsincan** [primefaces/primefaces#4681]
Reported By PRO User;
> When virtualScroll is used and frozen columns is used. 
Then when setting the selectedRow to a row outside of the scrollview from backing java bean. And then scrolling the selected row into view, the selected row is missing the aria-selected="true" attribute.
A manual mouseclick is needed and then the attribute is set correctly.
This only happens in the combination of virtualScroll, frozen columns and selected row set from java bean and then scrolled into view.
